### ISSUE TITLE
v2.1.2 improperly packaged on bower and general release issues

### ISSUE THREAD

**Issue by parris** [DmitryBaranovskiy/raphael#838]
This problem is a bit interesting. We noticed that this sha fixed an issue we were having:
https://github.com/DmitryBaranovskiy/raphael/commit/b8a34e5642994665fd482b5ddde403cf31d90a56

We tried a bower install 2.1.2 and noticed a version mismatch. Unpacking the tar ball revealed that the changes from that commit don't exist.

We tried to checkout v2.1.2 then we realize we were checking out the tag and not the branch. This sort of makes sense except the sha for the 2.1.2 tag is in master and missing some commits as compared to the branch.

In short, there is no way to get those changes and relying on a sha not tied to tag will result in random build failures.

Suggestions:
- Name tags and branches different things
- Branches should be used to fork off major/minor releases
- Patch versions should be created as a branch moves forward
- Publish to bower and npm as releases come out
- The npm version is out of sync with the bower version (I don't think you guys are maintaining the npm version but you probably should be).

**Comment by tomasAlabes** [comment]
All the packages are done now. Npm is updated and you can use whatever you want from bower.
